apisix/core/request.lua

```diff
@@ -334,6 +335,26 @@ function _M.get_body(max_size, ctx)
 end
 
 
+function _M.get_request_body_table()
+    local body, err = _M.get_body()
+    if not body then
+        return nil, { message = "could not get body: " .. (err or "request body is empty") }
+    end
+
+    body, err = body:gsub("\\\"", "\"") -- remove escaping in JSON
```

Comment:
I still don't understand the need for it. Can you provide a specific example? For example, curl or postman?

Reply:
hmm now I can no longer reproduce this issue. Maybe I was doing something wrong before.